doc/website/getting-started/installation.md

```diff
@@ -64,103 +64,113 @@ QNX SDP 7.0 and 7.1 are supported (shipping with gcc 5.4 and gcc 8.3 respectivel
 We provide generic QNX SDP 7.0 toolchain files for ARM_64 and X86_64 in `./tools/toolchains/qnx` ([Direct Link](https://github.com/eclipse-iceoryx/iceoryx/tree/master/tools/toolchains/qnx)).
 
-ARM_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_aarch64le.cmake`<br>
-X86_64: `./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_x86_64.cmake`
+ARM_64: 
+```bash
+./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_aarch64le.cmake
+```
+X86_64: 
+```bash
+./tools/iceoryx_build_test.sh -t /home/user/toolchains/qnx/qnx_sdp70_x86_64.cmake
+```
 
 !!! attention
     Please make sure the directory `/var/lock` is created by your QNX image
 
 ## Build with CMake
 
 !!! note
-    Requires CMake version 3.5 or higher. Building from CMake is the preferred way, for more complex actions like a coverage scan
+    Building from CMake is the preferred way, for more complex actions like a coverage scan
     is a script available (see chapter below).
 
 The `CMakeLists.txt` from `iceoryx_meta` can be used to easily develop iceoryx with an IDE.
 
 1. Clone the repository
 
-        git clone https://github.com/eclipse-iceoryx/iceoryx.git
+    ```sh
+    git clone https://github.com/eclipse-iceoryx/iceoryx.git
+    ```
 
 2. Generate the necessary build files
 
-        cd iceoryx
-        cmake -Bbuild -Hiceoryx_meta
-        # when you have installed external dependencies like ncurses you have to add them
-        # to your prefix path
-        cmake -Bbuild -Hiceoryx_meta -DCMAKE_PREFIX_PATH=$(PWD)/build/dependencies/
-
+    ```bash
+    cd iceoryx
+    cmake -Bbuild -Hiceoryx_meta
+    # when you have installed external dependencies like ncurses you have to add them
+    # to your prefix path
+    cmake -Bbuild -Hiceoryx_meta -DCMAKE_PREFIX_PATH=$(PWD)/build/dependencies/
+    ```
     !!! tip
         To build all iceoryx components add `-DBUILD_ALL` to the CMake command
 
 3. Compile the source code
-
-        cmake --build build
-
+    
+    ```bash
+    cmake --build build
+    ```
     !!! tip
         You can fasten up the build by appending `-j 4` where 4 stands for the number of parallel build processes.
         You can choose more or less depending on your available CPU cores on your machine.
 
 4. Install to system
 
     Mac:
-
-        cmake --build build --target install
-
+    ```bash
+    cmake --build build --target install
+    ```
     Linux:
-
-        sudo cmake --build build --target install
-
+    ```bash
+    sudo cmake --build build --target install
+    ```
     !!! tip
         The installation directory is usually left at its default, which is `/usr/local`
 
     !!! note
-        iceoryx is build in Release mode with `-O3` optimization by default. If you want to have     debug symbols please
+        iceoryx is build in Release mode with `-O3` optimization by default. If you want to enable debug symbols please
```

Comment:
Why do we use `-O2` in `tools/toolchains/qnx/qnx_sdp70_common.cmake` and not `-O3`?

Reply:
No reason. Was just some generic file. O3 is fine of course!

Reply:
is now set in the toolchain file